specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2021-01-01-preview/AuthorizationRules.json

```diff
@@ -0,0 +1,1100 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ServiceBusManagementClient",
+    "description": "Azure Service Bus client",
+    "version": "2021-01-01-preview"
```

Comment:
@apraovjr In this PR, I see you spread base files and new changes are mixed/spread across commits and it makes harder to review the delta changes. While adding new version the recommendation is to copy the existing versions into new directory structure for first commit and then push new changes in subsequent commits. Please raise a new PR as suggested for faster review.